pkg/operator/ceph/cluster/rbd/spec_test.go

```diff
@@ -53,11 +53,12 @@ func TestPodSpec(t *testing.T) {
 			},
 		},
 		metav1.OwnerReference{},
+		"/var/lib/rook/",
 	)
 	daemonConf := daemonConfig{
 		DaemonID:     "a",
 		ResourceName: "rook-ceph-rbd-mirror-a",
-		DataPathMap:  config.NewDatalessDaemonDataPathMap(),
```

Comment:
Again, I would argue that this hasn't stopped being a dataless daemon. The daemon itself still stores none of its own data, nor does it read data off the disk from a default dir as far as I can tell. I think it's important to distinguish this case because it is a thing that is distinctly different for rbd-mirror versus all the other Ceph daemons. `NoData` is indicating the fact that this daemon doesn't have a data path in the container, by the way. Not having a data dir seems to me to have implications as to what init containers are feasibly able to do before the daemon starts.